README.md

```diff
@@ -1 +1,113 @@
-# To-Do-list
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+  <h3><b>To Do List</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+**[To Do List]** is a project which adds/removes tasks to the list.<br />
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+ - HTML 
+ - CSS
+ - JAVASCRIPT (ES6)
+ - WEBPACK
+
+### Key Features <a name="key-features"></a>
+
+- **[Used ES6 Javascript]**
+- **[Used Webpack]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo link](https://zhorabay.github.io/To-Do-list/)
```

Comment:
- [ ] Kindly fix this live demo link or remove it.

- [ ] You seem to have gone ahead and implemented all the features of the entire to-do list app. Let's stick to the project specifications. There should be a hardcoded array of tasks with each item having `index`, `description`, and `completed` keys. There should be a function that iterates over these tasks and render them on the HTML page based on the design.

  This is a requirement for this very project and must be met before moving ahead. While you are free to add any features you wish the basic requirements should still be met.